runtime-config-linux.md

```diff
@@ -146,7 +146,7 @@ For more information, see the [kernel cgroups documentation](https://www.kernel.
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
 `cgroupsPath` is expected to be relative to the cgroups mount point.
-If not specified, cgroups will be created under '/'.
+If not specified or specified as empty string, implementations can define the default cgroup path.
```

Comment:
On Tue, Nov 17, 2015 at 05:43:12PM -0800, Vish Kannan wrote:
> > -If not specified, cgroups will be created under '/'.
> > +If not specified or specified as empty string, implementations can define the default cgroup path.
> 
> How about rephrasing as `If cgroups path is not specified, behavior
> is undefined`?

The behavior is defined: the runtime will create a new cgroup, add the
container process to it, and perform any resource allocation specified
by ‘resources’.  The only thing that's not defined when cgroupsPath is
not specified is the name of the cgroup that the runtime uses.  So I'd
rather go back to the original phrasing, where the only unspecified
behavior is how the runtime picks the default name used when
cgroupPath is unset.